icu4c/source/data/xml/brkitr/de.xml

```diff
@@ -6,9 +6,9 @@
 
  See http://bugs.icu-project.org/trac/ticket/10750#comment:5 - this file is needed to emit break iterator rules properly.
 -->
-<!DOCTYPE ldml SYSTEM "http://www.unicode.org/repos/cldr/trunk/common/dtd/ldml.dtd"
+<!DOCTYPE ldml SYSTEM "https://raw.githubusercontent.com/unicode-org/cldr/master/common/dtd/ldml.dtd"
 [
-   <!ENTITY % icu SYSTEM "http://www.unicode.org/repos/cldr/trunk/common/dtd/ldmlICU.dtd">
+   <!ENTITY % icu SYSTEM "https://raw.githubusercontent.com/unicode-org/cldr/master/common/dtd/ldmlICU.dtd">
```

Comment:
no…  move ldmlICU.dtd out of CLDR (new ticket required) and move it into the ICU source base. Then make this a relative link.

Reply:
But seems CLDR project has references to ldmlICU.dtd, and I am not sure if moving it out of CLDR does not break CLDR tools.
https://github.com/unicode-org/cldr/search?q=ldmlICU&type=Code
Can we file a separate ticket for this migration?

Reply:
CLDR needs to change, and ldml2icu is moving out of CLDR.

Reply:
CLDR's direct usage is:
- skip this DTD in certain processing
- a test case

that's it.

Reply:
Sorry for late reply.

I have cloned the file into ICU project now, and filed https://unicode-org.atlassian.net/browse/CLDR-13130.

Please take a look.